lib/logux.rb

```diff
@@ -65,16 +65,18 @@ def initialize(msg, meta: nil)
     config.render_backtrace_on_error = true
   end
 
-  def self.add(type, meta: {})
-    logux_add = Logux::Add.new
-    logux_meta = Logux::Meta.new(meta)
-    logux_add.call(type, meta: logux_meta)
+  def self.add(action, meta = Meta.new)
+    Logux::Add.new.call([[action, meta]])
```

Comment:
Do we need to pack `meta` to `Meta.new`? It is must easy to user to pass just plain object here

Reply:
It works in both ways: when provide {} and Meta class instance.
But here we need Meta.new to generate meta_id automatically